### DISCUSSION TITLE
How to display Dynamic data in Input ?

### DISCUSSION THREAD

**Discussion by Jahon93** [ant-design/ant-design#36079] [Q&A]
How can I display the dynamic value in Input which is coming from API?  
Input value is showing only hard-coded value. My code is :
  
![email](https://user-images.githubusercontent.com/55333953/174038166-35bca9d2-d42b-4511-a59d-96703e7d2159.PNG)

**Comment by yoyo837** [comment]
Remove `value`.

**Comment by yoyo837** [comment]
`initialValues` is probably what you want，See this:

https://ant.design/components/form-cn/#Form

**Reply by Jahon93** [reply]
Thank You !
